backend/aks/clean.sh

```diff
@@ -9,6 +9,9 @@ if [ -d "$BUILD_DIR" ]; then
     . .envrc
 
     if [ -d "cap-terraform/aks" ]; then
+
+	export KUBECONFIG="$(pwd)"/cap-terraform/aks/aksk8scfg
```

Comment:
Seems that after last lint fix, shellcheck SC2155 appears. This ought to fix it:

```suggestion
	KUBECONFIG="$(pwd)"/cap-terraform/aks/aksk8scfg
	export KUBECONFIG
```

Reply:
Thanks, applied the change.